roles/pulibrary.figgy/tasks/main.yml

```diff
@@ -183,3 +183,15 @@
     state: 'directory'
 - name: Install Google Cloud Authorization
   copy: src=files/{{rails_app_env}}-google_cloud_credentials.json dest=/home/{{deploy_user}}/cloud_config/google_cloud_credentials.json
+## Google Cloud Worker
+  - name: Install startup script for PubSub
```

Comment:
The indentation here needs to be aligned, otherwise I receive the following error:
```
ERROR! Syntax Error while loading YAML.
```

Reply:
Ah, yeah - fixed.